app/views/layouts/application.html.erb

```diff
@@ -37,12 +46,7 @@
     <aside>
       <article>
           <%= yield :aside %>
-      </article>
```

Comment:
The article is its own specific section so I think it has to be different from the footer